tools/build_profile.py

```diff
@@ -6,16 +6,19 @@
 import re
 import sys
 
+
 def PrintProfile(f):
-  prev_cmd = None
-  prev_timestamp = None
-  for line in f:
-    sr = re.match('\++ (\d+\.\d+) (.*)', line)
-    if sr:
-      timestamp, cmd = sr.groups()
-      if prev_cmd:
-        print '%.2f %s' % (float(timestamp) - float(prev_timestamp), prev_cmd)
-      prev_timestamp, prev_cmd = timestamp, cmd
+    prev_cmd = None
```

Comment:
can we set the tool for 2-char indent? That would be more consistent with our existing Python and habits.

Reply:
Consistency is definitely a valid reason for 2-char indentation. I can update for this but I just want to call out that it can be surprising for new contributors since [pep-8's recommendation](https://www.python.org/dev/peps/pep-0008/#indentation) is 4 spaces. :)